src/operations/expansions/inside.js

```diff
@@ -0,0 +1,49 @@
+/* Determine if the given points are inside the polygon.
+ *
+ * @param {Array} points - a list of points, where each point is an array with X and Y values
+ * @param {Array} polygon - a list of points, where each point is an array with X and Y values
+ * @return {Integer} 1 if all points are inside, 0 if some or none are inside
+ */
+const arePointsInside = (points, polygon) => {
+  let sum = points.reduce((acc, point) => {
+    return acc + isPointInside(point, polygon)
+  }, 0)
+  return sum === polygon.length ? 1 : 0
+}
+
+/* Determine if the given point is inside the polygon.
+ *
+ * @see http://paulbourke.net/geometry/polygonmesh/
+ * @param {Array} point - an array with X and Y values
+ * @param {Array} polygon - a list of points, where each point is an array with X and Y values
+ * @return {Integer} 1 if the point is inside, 0 if outside
+ */
+const isPointInside = (point, polygon) => {
+  let n = polygon.length
+  let counter = 0
+  let p1 = polygon[0]
+  for (let i = 1; i <= n; i++) {
+    let p2 = polygon[i % n]
+    if (point[1] > Math.min(p1[1], p2[1])) {
+      if (point[1] <= Math.max(p1[1], p2[1])) {
+        if (point[0] <= Math.max(p1[0], p2[0])) {
+          if (p1[1] !== p2[1]) {
+            let xinters = (point[1] - p1[1]) * (p2[0] - p1[0]) / (p2[1] - p1[1]) + p1[0]
+            if (p1[0] === p2[0] || point[0] <= xinters) {
+              counter++
+            }
+          }
+        }
+      }
+    }
+    p1 = p2
+  }
+
+  if ((counter % 2) === 0) return 0
+  return 1
+}
+
+module.exports = {
+  arePointsInside,
```

Comment:
I think we should only expose 'arePoints' variant, and make it auto convert single points to arrays (like we usually do, with toArray), what do you think ?

Reply:
```
const toArray = data => {
  if (!data) return []
  if (data.constructor !== Array) return [data]
  return data
}
```
toArray() won't work as each point is an array as well.
there could be a check for array of array.

Reply:
Reversing the parameters might be interesting... inside(polygon, ...points)